en/api/action.md

```diff
@@ -92,6 +92,28 @@ dispatchAction({
 
 **EVENT:** [legendselectchanged](~events.legendselectchanged)
 
+### legendAllSelect(Action)
+Selects all legends.
+
+```js
+dispatchAction({
+    type: 'legendAllSelect'
+})
+```
+
+**EVENT:** [legendselectall](~events.legendselectall)
+
+### legendInverseSelect(Action)
+Unselects all legends.
+

```

Comment:
I think it's inverse selection instead of unselecting. It may change to selected status after button clicked.

Reply:
Yes~ That's my fault. I'm sure you are right. :)